### PR TITLE
releases: update patch release to August cycle

### PR DESCRIPTION
**Pull request by cpanato**
releases: update patch release to August cycle

/assign @saschagrunert @puerco 
cc @kubernetes/release-engineering

### CODE REVIEW

File: data/releases/schedule.yaml [hunk 1]
```diff
@@ -44,13 +50,16 @@ schedules:
     - release: 1.20.1
       cherryPickDeadline: 2020-12-11
       targetDate: 2020-12-18
-      note: "Tagging Issue https://groups.google.com/g/kubernetes-dev/c/dNH2yknlCBA"
+      # note: "Tagging Issue https://groups.google.com/g/kubernetes-dev/c/dNH2yknlCBA"
```
Comment (saschagrunert) [inline data/releases/schedule.yaml:53]: What's the reason to comment that?

Comment (cpanato) [reply data/releases/schedule.yaml:53]: no reason, push by mistake while fixing the schedule-builder

Comment (cpanato) [reply data/releases/schedule.yaml:53]: fixed, thanks for the review @saschagrunert

**Review by saschagrunert — APPROVED** [review]
Thanks!

**Review by saschagrunert — APPROVED** [review]

**Pull request merged**